Add meta description to some pages (#3794)
wazuh/wazuh-documentation: source/user-manual/kibana-app/reference/custom-dashboard.rst

.. Copyright (C) 2021 Wazuh, Inc.
.. meta::

>   :description: Get to know the process of creating a set of custom visualizations using Kibana in Wazuh.

Reviewer: ```suggestion
  :description: Get to know the process of creating custom visualizations and dashboards using Kibana.
```
